File: orchestrator/states.py
"""Orchestrator state definitions."""

from __future__ import annotations

from enum import Enum


class OrchestratorState(str, Enum):
    """States for the orchestrator state machine."""

    IDLE = "idle"
    PLANNING = "planning"
    EXECUTING = "executing"
    REVIEWING = "reviewing"
    RESPONDING = "responding"

File: src/orchestrator/states.ts
/** Orchestrator 状态枚举 */
export const OrchestratorState = {
  IDLE: 'idle',
  PLANNING: 'planning',
  EXECUTING: 'executing',
  REVIEWING: 'reviewing',
  RESPONDING: 'responding',
} as const

export type OrchestratorState = typeof OrchestratorState[keyof typeof OrchestratorState]
